src/components/Selector/Selector.js

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
 
-export default function Selector() {
-  return <div></div>;
+export default function Selector(props) {
```

Comment:
Another way to do this is by destructuring setRace, setQuery, query etc from the props object instead of dot-notating into props every time you use one.